src/topojson/topojson.ml

```diff
@@ -698,24 +696,23 @@ module Make (J : Intf.Json) = struct
     let junction_table : (Position.t, Set.t) Hashtbl.t =
```

Comment:
We really want to know _every_ set of neighbours to begin with (let's get the algorithm correct before optimising). So I think we want something like: `(Position.t, Set.t list) Hashtbl.t`.